weechat_otr_test/test_command_finish.py

```diff
@@ -21,23 +21,23 @@ def after_setup(self):
         weechat_otr.ACCOUNTS['nick@server'] = account
 
     def test_finish_buffer(self):
-        weechat_otr.command_cb(None, 'server_nick2_buffer', 'finish')
+        self.call_command_cb(None, 'server_nick2_buffer', 'finish')
```

Comment:
These I would consider a bug in the current tests unrelated to 2.6 and could potentially be their own PR. They should have been using `weechat_otr.PYVER.to_str` like the other tests. 

How do you feel about calling to_str in each test instead of using a helper? It's less DRY but I think it makes what the test is doing clearer.